README.md

```diff
@@ -173,13 +173,24 @@ Settings related to Channel Logos.
     - `Prefer XMLTV` - Use the channel logo from the XMLTV file if available otherwise use the M3U logo.
 
 ### Media
-Media can be used to access a provider VOD library representing the hierarchy in a folder structure as PVR recordings. An M3U entry can denote that it's media by either having an M3U property of `EXT-X-PLAYLIST-TYPE` set to VOD or for the specific entry to have `media`, `media-dir` and `media-size` attributes. More detail on these can be found in [Supported M3U and XMLTV elements](#supported-m3u-and-xmltv-elements).
+Media entries can be used to access the Video-On-Demand (VOD) library provided by IPTV services. The IPTV media library can be organized either as plain list of channels or the hierarchy tree similar to Kodi's media library (folders and PVR recordings).
```

Comment:
We shouldn't reference kodi media library here. Users should know it means PVR recordings.